src/lib/pages/publish-module/components/leaflet/metadata.ts

```diff
@@ -0,0 +1,69 @@
+import { structToString } from "./utils";
+
+const indent = " ".repeat(4);
```

Comment:
Can we utilize `/t` instead of `" ".repeat(4)`?

Reply:
doesn't provide the same indent as using blank space and its consistent with structToString fn